js/infobar.js

```diff
@@ -253,6 +270,18 @@
     InfoBar.prototype.oncancel = {};
 
     /**
+     * Attempts to store the disabled status of the specified InfoBar in sessionStorage
+     * @param {string} prefName - The stored preference name for this InfoBar
+     */
+    InfoBar.prototype.setDisabledStatus = function(prefName) {
+        try {
+            sessionStorage.setItem(prefName, 'true');
+        } catch (ex) {
+            console.log('oncancel: Error while setting didabled status of InfoBar in sessionStorage');
```

Comment:
@jpetto What do you think? Just remove all of the `console.log` statements or, keep some of them?